src/components/List/List.js

```diff
@@ -1,3 +1,37 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import './list.css'
```

Comment:
please change to `import './List.css'` (since your file is capitalized) -- this breaks Storybook

Reply:
Updating the file name itself to list.css that I thought was done before.